client/login/wp-login/index.jsx

```diff
@@ -130,15 +63,15 @@ export class Login extends React.Component {
 			</a>
 		);
 
-		const showMagicLoginLink = magicLoginEnabled && ! magicLoginView && ! twoFactorAuthType && (
+		const showMagicLoginLink = isEnabled( 'magic-login' ) && ! twoFactorAuthType && (
```

Comment:
`showMagicLoginLink` sounds like an action, but it can eval to an actual element tree. I suggest `magicLoginLink`.